site/_posts/2021-11-08-how-to-authenticate-your-react-app.md

```diff
@@ -0,0 +1,1117 @@
+---
+layout: blog-post
+title: How to use OAuth to Add Authentication to Your React App
+description: In this tutorial, you’ll learn how to implement secure login/logout process with React and FusionAuth
+author: Akira Brand
+image: blogs/fusionauth-example-react-2021/how-to-use-oauth-to-add-authentication-to-your-react-app-header-image.png
+category: blog
+tags: tutorial-react tutorial-javascript client-react client-javascript tutorial-integration
+excerpt_separator: "<!--more-->"
+---
+*Please Note:* This tutorial reworks an earlier tutorial on implementing OAuth in React, which can be found [here](/blog/2020/03/10/securely-implement-oauth-in-react/).
```

Comment:
I would add this as a callout (see the laravel post for an example) and add it below the <!-- more --> tag, which controls what shows up on the blog index page.